### PR TITLE
add logs in clean history command

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**

### CODE REVIEW

File: ecommerce/core/management/commands/clean_history.py [hunk 1]
```diff
@@ -71,8 +71,12 @@ def handle(self, *args, **options):
...
                 history_batch = qs.values_list('id', flat=True)
+                batch_list = list(history_batch)
+
                 with transaction.atomic():
-                    model.history.filter(pk__in=list(history_batch)).delete()
-                    time.sleep(sleep_time)
+                    model.history.filter(pk__in=batch_list).delete()
 
+                time.sleep(sleep_time)
+                message = 'Deleted {}s from pk: {} to pk: {}'.format(model.__name__, batch_list[-1], batch_list[0])
```
Comment (clintonb) [inline ecommerce/core/management/commands/clean_history.py]: Ditch `str.format()` and use the logger's built-in interpolation.

```python
logger.info('Deleted instances of %s with PKs %d and %d', model.__name__, batch_list[0], batch_list[-1])
```

File: ecommerce/core/management/commands/clean_history.py [hunk 2]
```diff
@@ -71,8 +71,12 @@ def handle(self, *args, **options):
             qs = qs[:batch_size]
             while qs.exists():
                 history_batch = qs.values_list('id', flat=True)
+                batch_list = list(history_batch)
```
Comment (clintonb) [inline ecommerce/core/management/commands/clean_history.py]: There is no need for a new variable: `history_batch = list(qs.values_list('id', flat=True))`

**Review by feanil — APPROVED** [review]
This is an improvement, not sure if this will fix the issue that caused it to hang but will hopefully give us some more info and reduce the number of things inside the atomic block.

File: ecommerce/core/management/commands/clean_history.py (hunk 1 above)
Comment (clintonb) [reply ecommerce/core/management/commands/clean_history.py]: Move the log message into the transaction block.

**Review by clintonb — CHANGES_REQUESTED** [review]

File: ecommerce/core/management/commands/clean_history.py [hunk 3]
```diff
@@ -70,9 +70,14 @@ def handle(self, *args, **options):
...
             while qs.exists():
-                history_batch = qs.values_list('id', flat=True)
+                history_batch = list(qs.values_list('id', flat=True))
+
                 with transaction.atomic():
-                    model.history.filter(pk__in=list(history_batch)).delete()
-                    time.sleep(sleep_time)
+                    model.history.filter(pk__in=history_batch).delete()
+                    logger.info(
+                        'Deleted instances of %s with PKs %d and %d',
```
Comment (clintonb) [inline ecommerce/core/management/commands/clean_history.py]: with PKs between

**Review by clintonb — APPROVED** [review]
Update the commit message to follow our best practices: https://openedx.atlassian.net/wiki/spaces/OpenDev/pages/161880915/Git+Best+Practices. Use the previous ticket number.

**Comment by ahsan-ul-haq** [comment]
@clintonb made the suggested changes

**Review by clintonb — APPROVED** [review]

**Pull request merged**